PurchasesCoreSwift/Logging/Strings/AttributionStrings.swift

```diff
@@ -9,6 +9,17 @@
 import Foundation
 
 @objc(RCAttributionStrings) public class AttributionStrings: NSObject {
-    @objc public var instance_configured_posting_attribution: String { "There is an instance configured, posting attribution." }
-    @objc public var no_instance_configured_caching_attribution: String { "There is no instance configured, caching attribution." }
+    @objc public var appsflyer_id_deprecated: String { "The parameter key rc_appsflyer_id is deprecated. Pass networkUserId to addAttribution instead." } //warn
```

Comment:
the one thing that feels off here is having to add a comment at the end. 
Comments are not code, so it's easy for devs to mess up and mix-and-match. They're also subject to getting outdated quickly by refactors. Furthermore, we're duplicating the places we tie the string to the log intent - once here, once when we actually use the strings. 

To solve this, I think there's two approaches we could take: 
1. create a struct or class or some structure that holds both the string and the log intent, so that they're tied together in code. We'd refactor some of the log methods (or add new ones) that take in an instance of the new struct, and would get the string and the log intent from it
2. just keep the association between the message and the log intent at the callsite. 
I think `2.` is easier, both when creating the log messages and when implementing the first time now

Reply:
to clarify further, my suggested approach here would be to just remove the comments, and we can decide which log intent to use when calling the `Logger` methods

Reply:
This was mainly a guide for myself to track which log method to call for them and comments were there in this review to check off the log message and associated LogIntent. Once this is checked off, I would remove the comment after replacing the current log methods parameters with the string.